projects/views.py

```diff
@@ -312,6 +312,15 @@ def notify_join_request_user(django_request, join_request):
         )
 
 
+def set_budget_for_user_in_project(user, project, target_budget, enforced_by):
+    charge_budget, created = ChargeBudget.objects.get_or_create(
```

Comment:
I'm forgetting how get_or_create works, but will this create a new object if a different manager sets the budget, since enforced_by will be different?